pkg/markdown/markdown_test.go

```diff
@@ -1,38 +1,103 @@
 package markdown
 
 import (
+	"fmt"
+	"os"
+	"path/filepath"
 	"testing"
 
+	"github.com/cli/go-gh/v2/pkg/accessibility"
 	"github.com/stretchr/testify/assert"
 )
 
+// glamour theme colors found at https://github.com/charmbracelet/glamour/tree/master/styles
+const (
+	glamourLightH2Color     = "27"
+	glamourDarkH2Color      = "39"
+	customH2Color           = "61"
+	magentaEscapeCode       = "35"
+	brightMagentaEscapeCode = "95"
```

Comment:
Is it important to comment or modify the constant names to reflect which color depth these codes relate to?

- `magentaEscapeCode` and `brightMagentaEscapeCode` in this case are both [3-bit / 4-bit codes](https://en.wikipedia.org/wiki/ANSI_escape_code#3-bit_and_4-bit)
- `glamourLightH2Color`, `glamourDarkH2Color`, and `customH2Color` are [8-bit codes](https://en.wikipedia.org/wiki/ANSI_escape_code#8-bit)

Reply:
🤔 I'm not sure they're different for terminal escape codes, right? Like, the 4-bit codes are values 30-37 & 90-97, and they're the same number for 8-bit color sequences but are encoded differently as escape sequences. Maybe what would be better is extracting the whole escape sequence and renaming? So something like:

```suggestion
	glamourLightH2_8bitColorEscSeq     = "\x1b[38;5;27;"
	glamourDarkH2_8bitColorEscSeq      = "\x1b[38;5;39;"
	customH2_8bitColorEscSeq           = "\x1b[38;5;61;"
	magenta_4bitColorEscSeq       = "\x1b[35"
	brightMagenta_4bitColorEscSeq = "\x1b[95"
```

`#namesarehard`

Reply:
> and they're the same number for 8-bit color sequences but are encoded differently as escape sequences

I'm not sure I understand as [wiki](https://en.wikipedia.org/wiki/ANSI_escape_code#8-bit) shows that 8-bit color codes for ANSI colors are not the same:

```
ESC[38;5;⟨n⟩m Select foreground color      where n is a number from the table below
ESC[48;5;⟨n⟩m Select background color
  0-  7:  standard colors (as in ESC [ 30–37 m)
  8- 15:  high intensity colors (as in ESC [ 90–97 m)
 16-231:  6 × 6 × 6 cube (216 colors): 16 + 36 × r + 6 × g + b (0 ≤ r, g, b ≤ 5)
232-255:  grayscale from dark to light in 24 steps
```

Snippet from previous color contrast testing script displaying 8-bit ANSI colors where `(n)` goes from 0-15:

```shell
echo -e "   8 | Black               | ESC[48;5;0m   [\033[48;5;0m    \033[m] | ESC[38;5;0m   \033[38;5;0m andyfeller \033[m"
echo -e "   8 | Red                 | ESC[48;5;1m   [\033[48;5;1m    \033[m] | ESC[38;5;1m   \033[38;5;1m andyfeller \033[m"
echo -e "   8 | Green               | ESC[48;5;2m   [\033[48;5;2m    \033[m] | ESC[38;5;2m   \033[38;5;2m andyfeller \033[m"
echo -e "   8 | Yellow              | ESC[48;5;3m   [\033[48;5;3m    \033[m] | ESC[38;5;3m   \033[38;5;3m andyfeller \033[m"
echo -e "   8 | Blue                | ESC[48;5;4m   [\033[48;5;4m    \033[m] | ESC[38;5;4m   \033[38;5;4m andyfeller \033[m"
echo -e "   8 | Magenta             | ESC[48;5;5m   [\033[48;5;5m    \033[m] | ESC[38;5;5m   \033[38;5;5m andyfeller \033[m"
echo -e "   8 | Cyan                | ESC[48;5;6m   [\033[48;5;6m    \033[m] | ESC[38;5;6m   \033[38;5;6m andyfeller \033[m"
echo -e "   8 | White               | ESC[48;5;7m   [\033[48;5;7m    \033[m] | ESC[38;5;7m   \033[38;5;7m andyfeller \033[m"
echo -e "   8 | Bright Black        | ESC[48;5;8m   [\033[48;5;8m    \033[m] | ESC[38;5;8m   \033[38;5;8m andyfeller \033[m"
echo -e "   8 | Bright Red          | ESC[48;5;9m   [\033[48;5;9m    \033[m] | ESC[38;5;9m   \033[38;5;9m andyfeller \033[m"
echo -e "   8 | Bright Green        | ESC[48;5;10m  [\033[48;5;10m    \033[m] | ESC[38;5;10m  \033[38;5;10m andyfeller \033[m"
echo -e "   8 | Bright Yellow       | ESC[48;5;11m  [\033[48;5;11m    \033[m] | ESC[38;5;11m  \033[38;5;11m andyfeller \033[m"
echo -e "   8 | Bright Blue         | ESC[48;5;12m  [\033[48;5;12m    \033[m] | ESC[38;5;12m  \033[38;5;12m andyfeller \033[m"
echo -e "   8 | Bright Magenta      | ESC[48;5;13m  [\033[48;5;13m    \033[m] | ESC[38;5;13m  \033[38;5;13m andyfeller \033[m"
echo -e "   8 | Bright Cyan         | ESC[48;5;14m  [\033[48;5;14m    \033[m] | ESC[38;5;14m  \033[38;5;14m andyfeller \033[m"
echo -e "   8 | Bright White        | ESC[48;5;15m  [\033[48;5;15m    \033[m] | ESC[38;5;15m  \033[38;5;15m andyfeller \033[m"
e
```

All that being said, let me catch up on the actual changes as I suspect I understand what you're getting at.